Assets/Scripts/LoseScreenOverlays/SaveScore.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEngine;
+using System;
+
+public class SaveScore : MonoBehaviour
+{
+    
+    // Start is called before the first frame update
+    void Start()
+    {
+        if (PlayerPrefs.HasKey("TopScores"))
```

Comment:
Refer to constants file for player prefs keys

Reply:
completed.